floyd/cli/auth.py

```diff
@@ -23,6 +23,8 @@ def login(token):
     floyd_logger.info("Please copy and paste the authentication token.")
     access_code = click.prompt('This is an invisible field. Paste token and press ENTER', type=str, hide_input=True)
 
+    access_code = access_code.strip()
```

Comment:
There is another `access_code.strip(" ")` a few lines below this. Can that be removed?

Reply:
yeah, I think so